numpy/core/tests/test_scalarmath.py

```diff
@@ -397,13 +397,13 @@ def test_iinfo_long_values(self):
             assert_(res == tgt)
 
         for code in np.typecodes['AllInteger']:
-            res = np.typeDict[code](np.iinfo(code).max)
+            res = np.sctypeDict[code](np.iinfo(code).max)
             tgt = np.iinfo(code).max
             assert_(res == tgt)
 
     def test_int_raise_behaviour(self):
         def overflow_error_func(dtype):
-            np.typeDict[dtype](np.iinfo(dtype).max + 1)
+            np.sctypeDict[dtype](np.iinfo(dtype).max + 1)
 
         for code in 'lLqQ':
```

Comment:
Tempting to just eliminate `typeDict` here:
```suggestion
            dtype(np.iinfo(dtype).max + 1)

        for code in [np.int_, np.uint, np.longlong, np.ulonglong]:
```

Reply:
Might as well, yes. See https://github.com/numpy/numpy/pull/17586/commits/58f24b2d63a1a4016a42f5e9e8f9dd5f88a81a15.